website/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/wgs.methods.md

```diff
@@ -32,6 +32,30 @@ Prior to variant calling, the variant calling interval list was split to enable
 
 The pipeline’s final outputs included metrics, validation reports, an aligned CRAM with index, and a reblocked GVCF containing variant calls with an accompanying index.
 
+## Detailed methods for the Functional Equivalence mode of the Whole Genome Germline Single Sample workflow
+
+Preprocessing and variant calling was performed using the WholeGenomeGermlineSingleSample 3.0.0 pipeline using Picard 2.23.8, GATK 4.2.2.0, and Samtools 1.11 with default tool parameters unless otherwise specified. All reference files are available in the public [Broad References Google Bucket](https://console.cloud.google.com/storage/browser/gcp-public-data--broad-references/hg38/v0). The pipeline follows the Functional Equivalence specification ([Regier et al., 2018](https://www.nature.com/articles/s41467-018-06159-4)).
```

Comment:
Can we link to this article instead?
https://gatk.broadinstitute.org/hc/en-us/articles/4410456501915-Functional-equivalence-in-DRAGEN-GATK

Reply:
Yes, I added it. Thank you!